From 15b0903148d0dc07dc3492f7689221370fab57d2 Mon Sep 17 00:00:00 2001
From: Zdenek Dohnal <zdohnal@redhat.com>
Date: Tue, 6 Jan 2026 14:06:08 +0100
Subject: [PATCH] Fix bootstrap script

It fixed two issues:

- AM_GNU_GETTEXT in AX_REQUIRE_DEFINED caused error during autoconf,
  because AM_GNU_GETTEXT is now forbidden to use without arguments. Remove
  the AX_REQUIRE_DEFINED - if AM_GNU_GETTEXT is missing, autoconf will
  fail later anyway
- AM_GNU_GETTEXT_VERSION and AM_GNU_GETTEXT were not found during
  autoconf - use AC_CONFIG_MACRO_DIR to tell autoconf to search in local
  generated directory m4

Fixes #416
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9398fd3e6..e32e5c216 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,8 +1,8 @@
 AC_INIT(system-config-printer, 1.5.18)
 AC_CONFIG_SRCDIR(system-config-printer.py)
 AM_INIT_AUTOMAKE([dist-xz dist-bzip2 subdir-objects 1.6])
+AC_CONFIG_MACRO_DIR([m4])
 AM_GNU_GETTEXT_VERSION(0.20)
-AX_REQUIRE_DEFINED([AM_GNU_GETTEXT])
 AM_GNU_GETTEXT([external])
 AM_PATH_PYTHON([3])
 PACKAGE="AC_PACKAGE_NAME"
